content/ko/docs/concepts/storage/windows-storage.md

```diff
@@ -56,7 +57,7 @@ content_type: concept
 [플러그인](/ko/docs/concepts/storage/volumes/#volume-types) 형태로 제공된다. 
 윈도우는 다음의 광역 쿠버네티스 볼륨 플러그인 클래스를 지원한다.
 
-* [`FlexVolume 플러그인`](/ko/docs/concepts/storage/volumes/#flexvolume-deprecated)
+* [`FlexVolume 플러그인`](/ko/docs/concepts/storage/volumes/#flexvolume)
```

Comment:
I prefer ‘`flexVolume` 플러그인’ because the linked page describes plugin with its name as title, but I still wonder why it’s ‘`FlexVolume plugins`‘ in English version, not ‘`FlexVolume` plugins’.
How do you think about apply these changes also on the English documents?